Source/WebCore/dom/Node.h

```diff
@@ -286,8 +286,8 @@ class Node : public EventTarget {
     // Returns the parent node, but null if the parent node is a ShadowRoot.
     ContainerNode* nonShadowBoundaryParentNode() const;
 
-    bool selfOrAncestorHasDirAutoAttribute() const { return hasNodeFlag(NodeFlag::SelfOrAncestorHasDirAuto); }
-    void setSelfOrAncestorHasDirAutoAttribute(bool flag) { setNodeFlag(NodeFlag::SelfOrAncestorHasDirAuto, flag); }
+    bool selfOrPrecedingNodesAffectsDirAuto() const { return hasNodeFlag(NodeFlag::SelfOrPrecedingNodesAffectsDirAuto); }
```

Comment:
Should be "affect" rather than "affects" for subject verb agreement. Applies to all the variations in the patch.

Reply:
Oh good point. Will fix.